pom.xml

```diff
@@ -49,6 +49,48 @@
 		</contributor>
 	</contributors>
 
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>enforce-rules</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<requireMavenVersion>
+									<version>3.3.9</version>
+								</requireMavenVersion>
+								<requirePluginVersions>
+									<message>Plugins need to be versioned!</message>
+								</requirePluginVersions>
+								<requireJavaVersion>
+									<version>[1.8.0-101,1.11.9999]</version>
```

Comment:
Would it suffice to override the [`scijava.jvm.version` and `scijava.jvm.build.version`](https://github.com/scijava/pom-scijava-base/blob/edb51943e8e6e9de1a709c9fa2c2cb505a36a935/pom.xml#L164-L167) properties, instead of adding this whole `<plugin>` block?

Reply:
I am playing with Travis right now. I have to commit and do a PR to have it catch my changes, but all of this right now is very temporary. It's just that I do not know how to test Travis in another way.